ui/src/components/DebuggerControls.tsx

```diff
@@ -1,7 +1,56 @@
 import styles from './DebuggerControls.module.css';
 
-const DebuggerControls: React.FC = () => (
-  <div className={styles['debugger-controls']}>Debugger Controls</div>
+interface Props {
+  onExecute: () => void;
+}
+
+const DebuggerControls: React.FC<Props> = ({ onExecute }) => (
+  <div className={styles['debugger-controls']}>
+    <button
+      onClick={onExecute}
+      className={styles['debugger-controls__button']}
+      type="button"
+      name="execute"
+    >
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="24"
+        height="24"
+        fill="none"
+        stroke="currentColor"
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        aria-hidden
+        className={styles['debugger-controls__icon']}
+      >
+        <path d="m5 3 14 9-14 9V3z" />
+      </svg>
+      Execute
+    </button>
+    <button
+      onClick={(): void => console.log('Play script to end!')}
```

Comment:
Leaving this behavior stubbed out for now. I imagine that this will get an actual implementation once we have the line-by-line highlighting working.